### ISSUE TITLE
Rewind function changes empty arrays in properties to empty objects

### ISSUE THREAD

**Issue by david-deepblocks** [Turfjs/turf#1784]
I have a GeoJSON where some values in the  `properties` object are empty arrays.
```javaScript
properties: { stuff: [], ... }
```
 The problem is that calling `turf.rewind` returns the GeoJSON with those particular values inside `properties` as an empty objects instead, which is not what one would expect. 

```javaScript
properties: { stuff: {}, ... }
```

Why are the values in `properties` being modified anyway?

**Comment by rowanwins** [comment]
Hi @david-deepblocks 

It looks like this is caused by the cloning operation [here](https://github.com/Turfjs/turf/blob/2810f3755c4e246f24149869e7fe9e9623b352fa/src/rewind/index.js#L37) and [here](https://github.com/Turfjs/turf/blob/2810f3755c4e246f24149869e7fe9e9623b352fa/src/clone/index.js#L67)

If you set 
````
rewind(polygon, {mutate: true})
````
it works as expected, although it looks like this is a bug that should be resolved.

**Comment by FG-avi** [comment]
I ran into this issue as well when using `transformTranslate`. I used the work around given, but would be nice if this was fixed since mutating might not be possible all the time.

**Comment by rowanwins** [comment]
Resolved in v6.2.0

See https://jsfiddle.net/cd1x8fv9/
